proto/alarm/v1/config.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+
+package alarm.v1;
+
+option go_package = "alarm/config/v1;alarmconfig";
+
+message SendAlarmConfiguration{
+    string config_hash = 1;
+}
```

Comment:
Talked with @stelfrag, could perhaps a random key sent with this message, that the agent would include back in AlarmConfiguration payload mitigate a bit the concern of malicious AlarmConfiguration payloads?